### DISCUSSION TITLE
Linux version (or lack thereof)

### DISCUSSION THREAD

**Discussion by tredI9100** [WilliamVenner/gmpublisher#102] [Help] [answered]
How do I use this on Linux? There's no deb, flatpakref, or AppImage, and no instructions for compiling. I'm completely on my own, and have almost 0 experience with compiling, except for SheepShaver on my old computer, which just gave me an error that I don't know how to decipher. I've also tried gmpublish from Windows GMOD in Wine, but it just said that Steam wasn't running, even though it was, probably because I have the native Linux version of Steam. I also tried gmosh, but I just got the error "Addon too new to use old API" when using it in-game.

**Comment by WilliamVenner** [comment, marked as answer]
I only bother compiling for Windows at the moment because the other builds are broken in their own special ways. Macos/Linux mouse input doesn't work for example. I think it's a bug in the ancient version of Tauri I'm using and upgrading would require a rewrite.

**Reply by tredI9100** [reply]
Fair enough.

**Comment by WilliamVenner** [comment]
#333
